lib/widgets.php

```diff
@@ -126,3 +126,58 @@ function gutenberg_legacy_widget_settings( $settings ) {
 	return $settings;
 }
 add_filter( 'block_editor_settings', 'gutenberg_legacy_widget_settings' );
+
+
+function create_wp_area_post_type() {
```

Comment:
Should it be prefixed with `gutenberg_` ? I'd suggest it, unless the plan is for this function to be ported upstream directly, which I don't think is how it would be adopted.

Prior art: https://github.com/WordPress/wordpress-develop/blob/23867a2d7748d911d36107d39a6ea36dc834f763/src/wp-includes/post.php#L13-L432